### ISSUE TITLE
Change formattedDateString format?

### ISSUE THREAD

**Issue by martinbean** [briannesbitt/Carbon#431]
Is there a way to set a “formatted” string format globally with Carbon? I’ve seen the `formattedDateString()` method, but the format isn’t configurable.

What I’d like to be able to do is set a format globally (i.e. `Carbon::setDateStringFormat('l, j F Y')`) and then when I request a formatted `Carbon` date string, it’s the same format across my application. The format of `getFormattedDateString()` isn’t desirable as it’s U.S. format and abbreviated.

A similar method for times would be great as well, so instead of retrieving times via `toTimeString()` I could set a global time format (`Carbon::setTimeStringFormat('g:i a'')`) and then call a method like `getFormattedTimeString()`.

Are there methods that will allow me to do this presently?

**Comment by briannesbitt** [comment]
Check out Carbon::setToStringFormat().

It overwrites the default toString() format to whatever you want.

**Comment by martinbean** [comment]
@briannesbitt Would that not—for want of a better phrase—bugger up usage in Eloquent models in Laravel? (I think Eloquent casts Carbon instances to a string to get the YYYY-MM-DD HH:MM:SS format).

**Comment by briannesbitt** [comment]
I doubt they would rely on a toString behaviour for that.... seems brittle to me and the underlying DateTime didn't really provide anything useful as toString if I remember right.

I am not a laravel user but I have used eloquent and don't remember having any issues.  I did see a grammar function that returned a format string of "return 'Y-m-d H:i:s';" used for `Carbon::format()` so I think its probably fine.
